src/components/HotspotConfigurationPicker.tsx

```diff
@@ -15,6 +15,37 @@ import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
 import { useColors } from '../theme/themeHooks'
 import { Antenna, Antennas } from '../constants/antennas'
 
+export type AntennaId =
+  | 'helium_us'
+  | 'helium_eu'
+  | 'rak_hotspot_us'
+  | 'rak_hotspot_eu'
+  | 'nebra_outdoor'
+  | 'nebra_indoor'
+  | 'bobcat'
+  | 'syncrobit_us'
+  | 'syncrobit_eu'
+  | 'rak_custom'
+  | 'longapone_eu'
+  | 'finestra_us'
```

Comment:
You need to move this to https://github.com/helium/hotspot-app/blob/main/src/constants/antennas.ts#L13